db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_06_05_133804) do
+ActiveRecord::Schema[7.0].define(version: 2023_05_25_081252) do
```

Comment:
It doesn't really matter but I thought that we use the later date when resolving conflicts on the db version because it's an indicator of the version which should be the latest after all migrations ran.

Do you have a different perspective here?

Reply:
Oh, yes I didn't see it as a conflict. I will remove the commit. And hopefully a retest will resolve those flaky specs..